Dockerfile

```diff
@@ -25,7 +25,16 @@ RUN echo "AUTOSTART=none" > /etc/default/openvpn \
 COPY package.json package-lock.json /usr/src/app/
 RUN npm install --unsafe-perm --production && npm cache clean --force 2>/dev/null
```

Comment:
This should be:
```suggestion
RUN npm ci --unsafe-perm --production && npm cache clean --force 2>/dev/null

Reply:
Is the install not already clean by virtue of being in a docker image? We've only added the `package{,-lock}.json` files by this point.